libcudacxx/test/libcudacxx/std/numerics/c.math/logarithms.pass.cpp

```diff
@@ -101,9 +101,39 @@ __global__ void test_global_kernel(float* value)
   test(*value);
 }
 
+template <class T>
+__host__ __device__ _LIBCUDACXX_CONSTEXPR_BIT_CAST void test_constexpr(T value)
+{
+  test_ilogb<T>(value);
```

Comment:
Reminder to extent those tests a bit. They are really not sufficient, but the weekend and a vacation are coming up so...